x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/field_select.tsx

```diff
@@ -45,6 +44,10 @@ export interface FieldSelectProps extends EuiComboBoxProps<EuiComboBoxOptionOpti
   markAllFieldsCompatible?: boolean;
 }
 
+const DEFAULT_COMBOBOX_WIDTH = 305;
+const COMBOBOX_PADDINGS = 90;
```

Comment:
I see that you're always subtracting 90 for padding: seems brittle to me, for example if EUI changes something this could break. Not sure it's worth optimizing for but is there an alternative?

Reply:
I agree, but the problem is that I prefer to have one place where I take the ref from than to do it internally in every option in the select list.